scripts/album.js

```diff
@@ -1,7 +1,26 @@
 var setSong = function(songNumber){
+  if (currentSoundFile) {
+    this.stop();
+  }
   currentlyPlayingSongNumber = parseInt(songNumber);
-  currentSongFromAlbum = currentAlbum.songs[songNumber-1]; //i tried to parseInt this but it's an object, not an int
+  currentSongFromAlbum = currentAlbum.songs[songNumber-1];
+  //new buzz sound object
+  //new settingds object w/2 properties
+  currentSoundFile = new buzz.sound(currentSongFromAlbum.audioUrl, { 
```

Comment:
For some reason for awhile this was causing an error because currentSongFromAlbum wasn't getting set - console.logging it was returning "object object". Not sure what i changed to get rid of it but it seems fine now.